packages/unified-latex-builder/package.json

```diff
@@ -15,19 +15,23 @@
     ],
     "exports": {
         ".": {
+            "_bundle": "./index.ts",
             "import": "./dist/index.js",
             "require": "./dist/index.cjs"
         },
         "./*js": "./dist/*js",
         "./*": {
+            "_bundle": "./*.ts",
             "import": "./dist/*.js",
             "require": "./dist/*.cjs"
         }
     },
     "scripts": {
         "build": "npm run clean && mkdirp ./dist && npm run compile",
-        "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
-        "compile": "tsc -b tsconfig.json & node build.js & for job in `jobs -p`; do wait ${job}; done",
+        "clean": "rimraf ./dist",
+        "compile:tsc": "tsc",
+        "compile:esm_and_cjs": "node build.js",
+        "compile": "run-p compile:tsc compile:esm_and_cjs",
```

Comment:
This is great. I didn't know about this utility!

Reply:
My other PR uses `wireit` to do this, but it turns out the build didn't get sped up at all...(it was slowed down a little bit). You can look at it and see if I did something obviously wrong...